packages/app/src/app/pages/Sandbox/OnBoarding/index.tsx

```diff
@@ -23,9 +23,10 @@ const OnBoarding = () => {
   const [visibility, setVisibility] = useState(false);
   const [currentIndex, setCurrentIndex] = useState(0);
   const [sliderPosition, setSliderPosition] = useState(0);
+  const { browser } = useEffects();
 
   useEffect(() => {
-    const shouldOnboarding = localStorage.getItem('should-onboarding-user');
+    const shouldOnboarding = browser.storage.get('should-onboard-user');
```

Comment:
We can change the const name too :)

Reply:
Indeed! [29b4ca9](https://github.com/codesandbox/codesandbox-client/pull/7421/commits/29b4ca9359e90470cae2dc3a5c5b6f4f6695bb5b)